Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[workspace]
+members = [
+    "apps",
+    "deepstream/pravega_protocol_adapter",
+    "gst-plugin-pravega",
+    "pravega-video",
+    "pravega-video-server",
+    "integration-test"
```

Comment:
Keep this list in alphabetical order.

Reply:
My brain keeps wanting to put these in dependency order! 🤣